landoui/templates/partials/layout.html

```diff
@@ -13,7 +13,7 @@
 {% include "partials/navbar.html" %}
 <section class="hero is-light is-bold">
   <div class="container">
-    <p class="subtitle">{% block banner_text %}{% endblock %}</p>
+    <div class="subtitle"></div>
```

Comment:
I think that gradient is a nice top nav border but I don't like it as a page heading -- seems a bit Web 2.0-ish.